App/features/signup.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Signup
+  In order to acces the application
+  As a visitor user
+  I need to be able to create an account
+
+  Background:
+    Given I am on /signup
+
+  Scenario Outline: Sign-up the application failed
+    And I fill in the input Username with "<username>"
+    And I fill in the input Email with "<email>"
+    And I fill in the input Password with "<password>"
+    And I fill in the input Confirm password with "<confirm_password>"
+    When I submit the form
+    Then I should be redirect to "/signup"
```

Comment:
T'as pas un flash message qui te dit que ta création de compte a échouée ?

Reply:
J'ai des messages d'erreurs qui sont testés :
` /**
     * @Then I should see the message :message
     */
    public function iShouldSeeTheMessage(string $message): void
    {
        $fieldValue = $this->getSession()->getPage()->find('css', '.authentication_form_error > ul > li');
        if ($fieldValue->getText() != $message) {
            throw new \Exception("Message not found");
        }
    }`